lib/src/location_picker_field.dart

```diff
@@ -28,6 +28,11 @@ class _BaseFormField<T> extends StatefulWidget {
   final FocusNode? focusNode;
   final bool expands;
   final Function(FormFieldState<T> field) onRemove;
+  final Icon locationIcon;
```

Comment:
create a class holds the new properties `MapViewConfig` 
and add these new props in `OpenMapSettings` so that it can be configured globally

Reply:
I think that properties that refer to the map style should not go in the global configuration, because it is possible that you want to have two maps with different styles in the same app

Reply:
I think would be a good idea encapsulating everything in a map style class (something like `MapViewConfig`). This would give the possibility of having multiple styles.